config/config.yaml

```diff
@@ -33,7 +33,7 @@ redis:
   redisNetwork: "tcp"
 
 etcd:
-  address: 127.0.0.1:2380
+  address: 127.0.0.1:2379
```

Comment:
dont't change this etcd address. the 2380 of etcd is for micro service, and the 2379 of etcd if for milvus.